### PR TITLE
[MRG]Add contribution guide

### LINKED ISSUE #152: Add contribution guide and modify `CONTRIBUTING.md`
#### Software and Hardware Versions 
ModelCI v1.x.x, CUDA Version vx.x.x, GPU device used...

#### Problem description

The [Setup Environment part](https://github.com/cap-ntu/ML-Model-CI/blob/master/CONTRIBUTING.md#setup-environment) of  `CONTRIBUTING.md` is out of date,  and we could add a detailed contribute guide for new contributors with few experience on git to have a starting point

#### Other Information 
Things you tried, stack traces, related issues, suggestions on how to fix it...

### PR DESCRIPTION
**Pull request by univerone**
Closes #152 

#### Does this change require updates to documentation?
Yes

#### Before submitting this PR, please make sure you have:

- [x] run `python -m pytest tests/` and confirmed all checks still pass.
- [x] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -59,3 +59,98 @@ tests.
...
 We have some static checks when you filing a PR or pushing commits to the project, please make sure you can pass all the tests and make sure the coding style meets our requirements.
 
+## Contribution guide
+
+### 1. Clone your fork and configure the upstream
+
+Firstly, you have to fork this repo: <https://github.com/cap-ntu/ML-Model-CI> and create a local clone of this fork
+
+```shell
+git clone git@github.com:<YOUR-USERNAME>/ML-Model-CI.git
```
Comment (YuanmingLeee) [inline CONTRIBUTING.md:69]: better use https://github.com/....

**Comment by huangyz0918** [comment]
Thanks for the contribution! @univerone A reminder for open source habits, when you are still working on this PR, please tag `[WIP]`, once you think that is ready, tag `[MRG]` so that other contributors can start a detailed review. For more details, you can refer to the [sklearn contributing](https://scikit-learn.org/stable/developers/contributing.html).

branches in your fork should not be something named as `issue-number`, it will be really hard for you to index if there are many similar local branches.

**Comment by univerone** [comment]
> Thanks for the contribution! @univerone A reminder for open source habits, when you are still working on this PR, please tag `[WIP]`, once you think that is ready, tag `[MRG]` so that other contributors can start a detailed review. For more details, you can refer to the [sklearn contributing](https://scikit-learn.org/stable/developers/contributing.html).
> 
> branches in your fork should not be something named as `issue-number`, it will be really hard for you to index if there are many similar local branches.

Thanks for your advice! I'll change the branch name into a more descriptive one

File: CONTRIBUTING.md (hunk 1 above)
Comment (univerone) [reply CONTRIBUTING.md:69]: > better use https://github.com/....

Did that mean cloning repo by HTTPS is more recommended than by SSH

Comment (YuanmingLeee) [reply CONTRIBUTING.md:69]: Users need to setup ssh. Windows does not come with ssh, so clone with HTTPS is more convenient

**Review by YuanmingLeee — APPROVED** [review]
LGTM

**Review by HuaizhengZhang — APPROVED** [review]

**Pull request merged**